acceptance/framework/config/config.go

```diff
@@ -131,6 +132,25 @@ func (t *TestConfig) HelmValuesFromConfig() (map[string]string, error) {
 		setIfNotEmpty(helmValues, "global.openshift.enabled", "true")
 	}
 
+	setIfNotEmpty(helmValues, "connectInject.nodeSelector", "kubernetes.io/os: linux")
+	setIfNotEmpty(helmValues, "connectInject.apiGateway.managedGatewayClass.nodeSelector", "kubernetes.io/os: linux")
+	setIfNotEmpty(helmValues, "client.nodeSelector", "kubernetes.io/os: linux")
+	setIfNotEmpty(helmValues, "acls.nodeSelector", "kubernetes.io/os: linux")
+	setIfNotEmpty(helmValues, "global.acls.nodeSelector", "kubernetes.io/os: linux")
+	setIfNotEmpty(helmValues, "global.tls.nodeSelector", "kubernetes.io/os: linux")
+	setIfNotEmpty(helmValues, "meshGateway.nodeSelector", "kubernetes.io/os: linux")
+	setIfNotEmpty(helmValues, "server.nodeSelector", "kubernetes.io/os: linux")
+	setIfNotEmpty(helmValues, "syncCatalog.nodeSelector", "kubernetes.io/os: linux")
+	setIfNotEmpty(helmValues, "telemetryCollector.nodeSelector", "kubernetes.io/os: linux")
+	setIfNotEmpty(helmValues, "webhookCertManager.nodeSelector", "kubernetes.io/os: linux")
+	setIfNotEmpty(helmValues, "ingressGateways.defaults.nodeSelector", "kubernetes.io/os: linux")
+	setIfNotEmpty(helmValues, "terminatingGateways.defaults.nodeSelector", "kubernetes.io/os: linux")
+
+	if t.EnableWindows {
+		setIfNotEmpty(helmValues, "global.imageK8SWindows", t.ConsulK8SImage+":windows")
```

Comment:
Hey, windows tag is added because @curtbushko had asked us to use one variable and different tags for consulk8simage. We need both images here - linux as well as windows. Hence we added windows tag

Reply:
So the helm key would change in this case?

Reply:
we will sending image name without tag as a test flag and setting tags here to configure windows image or linux image

Reply:
thanks @curtbushko for the clarification

Reply:
have made the change